docs/sdks/uid2-sdk-ref-ios.md

```diff
@@ -27,22 +27,27 @@ import Link from '@docusaurus/Link';
 - [Android Initialization](#android-initialization)
 - [Code Samples](#code-samples) -->
 
-You can use the UID2 SDK for iOS to facilitate the process of establishing client identity using UID2 and retrieving advertising tokens on iOS devices.
+You can use the UID2 SDK for iOS and tvOS to facilitate the process of establishing client identity using UID2 and retrieving advertising tokens on iOS devices for bid stream use.
 
 The following iOS-related plugins, and associated documentation, are also available:
 
-| Purpose | Product/Documentation |
-| :--- | :--- |
-| To use the Google Mobile Ads (GMA) SDK to send [UID2 tokens](../ref-info/glossary-uid.md#gl-uid2-token) as [secure signals](https://support.google.com/admob/answer/11556288) in ad requests from iOS apps | [UID2 GMA Plugin for iOS Integration Guide](../guides/mobile-plugin-gma-ios.md) |
-| To use the Google Interactive Media Ads SDK for iOS to send [UID2 tokens](../ref-info/glossary-uid.md#gl-uid2-token) as [secure signals](https://support.google.com/admob/answer/11556288) in ad requests from iOS apps | [UID2 IMA Plugin for iOS Integration Guide](../guides/mobile-plugin-ima-ios.md) |
+| Purpose                                                                                                                                                                                                                 | Product/Documentation |
+|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| :--- |
+| To use the Google Mobile Ads (GMA) SDK to send [UID2 tokens](../ref-info/glossary-uid.md#gl-uid2-token) as [secure signals](https://support.google.com/admob/answer/11556288) in ad requests from iOS/tvOS apps         | [UID2 GMA Plugin for iOS Integration Guide](../guides/mobile-plugin-gma-ios.md) |
+| To use the Google Interactive Media Ads SDK for iOS to send [UID2 tokens](../ref-info/glossary-uid.md#gl-uid2-token) as [secure signals](https://support.google.com/admob/answer/11556288) in ad requests from iOS/tvOS apps | [UID2 IMA Plugin for iOS Integration Guide](../guides/mobile-plugin-ima-ios.md) |
+
+
+## tvOS Support
```

Comment:
if we do not want to mention tvOS anywhere else, we should at least keep this section

Reply:
decided to add this section and do not mention tvOS in other pages